src/algebraic_geometry/is_open_comap_C.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import algebraic_geometry.prime_spectrum
+import ring_theory.polynomial.basic
+/-!
+The morphism `Spec R[x] --> Spec R` induced by the natural inclusion `R --> R[x]` is an open map.
+-/
+
+open ideal polynomial prime_spectrum set integral_domain
+
+local attribute [instance] classical.prop_decidable
+
+variables {R : Type*} [comm_ring R] {P : ideal R}
+
+/-- If `P` is a prime ideal of `R`, then `R[x]/(P)` is an integral domain. -/
+lemma is_integral_domain_C_map_quotient (H : is_prime P) :
```

Comment:
It feels to me like these two lemmas should be in another file. Probably somewhere in `polynomial/something.lean`. One of those files probably already imports the definition of prime ideals.